devnet/devnet.json

```diff
@@ -1 +1,12 @@
-{ "composable": { "name": "picasso", "version": "2517dd16bf4a3470378699bcfec935ddc2020e0a", "hash": "15s2xg4hl44pdriwwpx33gq10kxzm4zhn7z36il8q2pwx3231r91", "revhash": "0ydd8hp8hm7cljviz0fvfhn253gwig45qpdigqr33cqj85ajbw4b" }, "polkadot": { "version": "0.9.16", "hash": "sha256-zDu0Tj7cSCER/ASxQmt+JCjoxLZc8UI8nYkrqXtveRU=" } }
+{
+    "composable": {
+        "name": "picasso",
+        "version": "2517dd16bf4a3470378699bcfec935ddc2020e0a",
+        "hash": "15s2xg4hl44pdriwwpx33gq10kxzm4zhn7z36il8q2pwx3231r91",
+        "revhash": "0ydd8hp8hm7cljviz0fvfhn253gwig45qpdigqr33cqj85ajbw4b"
+    },
+    "polkadot": {
+        "version": "0.9.16",
+        "hash": "sha256-zDu0Tj7cSCER/ASxQmt+JCjoxLZc8UI8nYkrqXtveRU="
+    }
+}
```

Comment:
Shouldn't we read the composable parameters using git instead of hardcoding?

Reply:
That is legacy and will be removed :)

Reply:
it is read already. but this file is here because not all migrated to nix. here is nix -> load json -> convert to nix. this is more of like tempale. and also nix fails to see not under git files until commited. so really yes. it should be removed.

Reply:
anyway, it should removed

Reply:
this json is used to allow nix evaluate check, more like json schema

Reply:
added todo in reader side